staubli_rx160_gazebo/launch/spawn_rx160.launch

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<launch>  
+
+  <!-- remap topics to conform to ROS-I specifications -->
+  <remap from="/arm_controller/follow_joint_trajectory" to="/joint_trajectory_action" />
+  <remap from="/arm_controller/state" to="/feedback_states" />
+  <remap from="/arm_controller/command" to="/joint_path_command"/>
+
+  <!-- send robot urdf to ROS param server -->
+  <include file="$(find staubli_rx160_gazebo)/launch/load_rx160.launch" />
+
+  <!-- push robot_description to factory and spawn robot in gazebo -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" 
+        output="screen" respawn="false"
+        args="-urdf
+              -param robot_description
+              -model rx160
+              -x 0.0
+              -y 0.0
+              -z 0.0
+              -R 0.0
+              -P 0.0
+              -Y 0.0" />
+
+    <!-- convert joint states to TF transforms for rviz, etc -->
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
+  output="screen">
+  </node>
+
+  <!-- init and start Gazebo ros_control interface -->
+  <include file="$(find staubli_rx160_gazebo)/launch/rx160_control.launch"/>
+  
+</launch>
```

Comment:
Your point makes sense. The way I'm doing at the moment means that I can have one launch file loading my Gazebo world, then spawning robots (and their controllers) under different namespaces in the world. There would need to be extra launch files for the controllers, matching the respective namespaces.

Multi-robot systems are usually neglected by most ROS packages, but I'll have a look around and see if I find a good example of how they separate responsibilities better.

Shall we separate spawning and control then? Or leave as it is for now?